checkov/cloudformation/checks/resource/aws/LambdaServicePermission.py

```diff
@@ -0,0 +1,35 @@
+from typing import List
+
+from checkov.cloudformation.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckCategories, CheckResult
+
+
+class LambdaServicePermission(BaseResourceCheck):
+    def __init__(self) -> None:
+        name = "Ensure that AWS Lambda function permissions delegated to AWS services are limited by SourceArn or SourceAccount"
+        id = "CKV_AWS_293"
+        supported_resources = ["AWS::Lambda::Permission"]
+        categories = [CheckCategories.GENERAL_SECURITY, ]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        properties = conf.get('Properties')
+        if properties is not None:
+            principal = properties.get('Principal')
+            principal_parts = principal.split('.')
+            try:
+                if principal_parts[1] == 'amazonaws' and principal_parts[2] == 'com':
+                    if properties.get('SourceArn') or properties.get('SourceAccount'):
+                        return CheckResult.PASSED
+                    else:
+                        return CheckResult.FAILED
+            except IndexError:
+                # Not a service principal, so pass.
+                return CheckResult.PASSED
+        return CheckResult.PASSED
```

Comment:
```suggestion
                return CheckResult.UNKNOWN
        return CheckResult.UNKNOWN
```
unknown fits better, because the check is not valid, if we don't find AWS principal.